R/create_phs_html.R

```diff
@@ -0,0 +1,62 @@
+#' create_phs_html
+#' @description Creates template `.qmd` file for a html report with PHS formatting.
+#'
+#' @param file_name Character string of name given to output `.qmd` file.
+#' @param ext_name Character string of extension folder name (default = "phs-html-quarto")
+#'
+#' @return Quarto document for html with PHS formatting.
+#' @export
+#' @examples
+#' \dontrun{
+#' create_phs_html()
+#' }
+
+create_phs_html <- function(file_name = NULL, ext_name = "phs-html-quarto") {
+  if (is.null(file_name)) {
+    stop("You must provide a valid file_name")
+  }
+
+  # check for available extensions
+  stopifnot("Extension not in package" = ext_name %in% c("phs-html-quarto"))
+
+  # check for existing _extensions directory
+  if (!file.exists("_extensions")) dir.create("_extensions")
+  message("Created '_extensions' folder")
+
+  # create folder
+  if (!file.exists(paste0("_extensions/", ext_name)))
+    dir.create(paste0("_extensions/", ext_name))
```

Comment:
Using `file.path` instead of `paste0` makes it safer on different OS's / file systems (Windows).
```suggestion
  if (!file.exists(file.path("_extensions", ext_name)))
    dir.create(file.path("_extensions", ext_name))
```